### DISCUSSION TITLE
API routes not getting documented if prefix set to empty

### DISCUSSION THREAD

**Discussion by haizad** [dedoc/scramble#655] [Q&A]
Hi, I am just started exploring this wonderful package, but currently stuck on displaying API routes in the generated documetation. My API endpoint not ends with /api , instead just like this http://localhost:80. 

What I tried so far is set `api_path` to empty string, and also configured `AppServiceProvider.php` file, but nothing have changes. :thinking: 

config/scramble.php
```
'api_path' => '',
```

bootstrap/app.php
```
return Application::configure(basePath: dirname(__DIR__))
    ->withRouting(
        web: __DIR__.'/../routes/web.php',
        api: __DIR__.'/../routes/api.php',
        apiPrefix: '',
        commands: __DIR__.'/../routes/console.php',
        health: '/up',
    )
```

app/Providers/AppServiceProvider.php
```
public function boot(): void
    {
        Scramble::routes(function (Route $route) {
            return Str::startsWith($route->uri, '/');
        });
    }
```

- I am using Laravel 11

Any help would be greatly appreciated. Thank you!

**Comment by romalytvynenko** [comment]
@haizad I believe you should drop that `Scramble::routes` config. Just setting `api_path` to blank string should be enough.
